madmimiofficial.php

```diff
@@ -114,32 +132,37 @@ public function register_shortcode() {
 	}
 
 	public function register_widget() {
+
 		register_widget( 'Mad_Mimi_Form_Widget' );
+
 	}
 
 	public function enqueue() {
 
+		$suffix = SCRIPT_DEBUG ? '' : '.min';
+		$rtl    = ! is_rtl() ? '' : '-rtl';
+
 		// main JavaScript file
-		wp_enqueue_script( 'mimi-main', plugins_url( 'js/mimi.js', __FILE__ ), array( 'jquery' ), MADMIMI_VERSION, true );
+		wp_enqueue_script( 'mimi-main', plugins_url( "js/mimi{$suffix}.js", __FILE__ ), array( 'jquery' ), MADMIMI_VERSION, true );
 
 		// datepicker JavaScript file
-		wp_enqueue_script( 'function', plugins_url( 'js/function.js', __FILE__ ), array( 'jquery' ), MADMIMI_VERSION, true );
+		wp_enqueue_script( 'function', plugins_url( "js/function{$suffix}.js", __FILE__ ), array( 'jquery' ), MADMIMI_VERSION, true );
```

Comment:
@EvanHerman Using `function` as the tag name is just asking for trouble. I also think this can be replaced by `wp_add_inline_script` because it's just a one-liner that instantiates the picker.